pkg/types/types.proto

```diff
@@ -46,3 +46,12 @@ message Route {
 	string source = 4;
 	uint32 scope = 5;
 }
+
+message ARPNeighbor {
+	IPAddress toIPAddress = 1;
+	string device = 2;
+	string lladdr = 3;
+	int32 state = 4;
+	int32 flags = 6;
```

Comment:
and where is 5 ?

Reply:
uh oh, missed this while deleting a field, will update.